README.md

```diff
@@ -206,6 +206,7 @@ The following pieces of container metadata are available to plugins in NRI:
         - cpuset memory
       - Block I/O class
       - RDT class
+      - Unified config map
```

Comment:
nit: I think we could be a bit more specific here, for instance by calling this `Unified cgroup v2 parameter map` or something to that effect ?